### PR TITLE
feat: modify zclclusterview popup to page

### PR DESCRIPTION
**Pull request by tbrkollar**

### CODE REVIEW

File: src/components/ZclClusterView.vue [hunk 1]
```diff
@@ -198,12 +207,7 @@ export default {
       this.tab = val
     },
   },
-  mounted() {
```
Comment (brdandu) [inline src/components/ZclClusterView.vue:201]: Why is this removed?

File: src/components/ZclClusterView.vue [hunk 2]
```diff
@@ -222,20 +226,10 @@ export default {
     setIndividualClusterFilterString(filterString) {
       this.$store.dispatch('zap/setIndividualClusterFilterString', filterString)
     },
-    onResize(size) {
```
Comment (brdandu) [inline src/components/ZclClusterView.vue:225]: Why is this removed?

File: src/components/ZclClusterView.vue [hunk 3]
```diff
@@ -222,20 +226,10 @@ export default {
...
-    },
-    calculateTableSize() {
-      this.tableHeight = '30px'
-      this.tableWidth = '500px'
-    },
   },
   data() {
     return {
       tab: 'attributes',
-      tableHeight: '30px',
```
Comment (brdandu) [inline src/components/ZclClusterView.vue:237]: What does this change?

File: src/components/ZclCreateModifyEndpoint.vue [hunk 4]
```diff
@@ -58,7 +58,7 @@ limitations under the License.
             class="col v-step-2"
             use-input
             :multiple="enableMultipleDevice"
-            :use-chips="enableMultipleDevice"
+            use-chips
```
Comment (brdandu) [inline src/components/ZclCreateModifyEndpoint.vue:61]: What does this do?

File: src/layouts/ZclLayout.vue
Comment (brdandu) [inline src/layouts/ZclLayout.vue:1]: Were the 3 .vue files that are deleted not used at all?

File: test/ui.test.js [hunk 5]
```diff
@@ -185,18 +183,7 @@ describe('Component mounting test', () => {
...
-    () => {
-      const wrapper = shallowMount(ZclClusterLayout, {
-        global: {
-          plugins: [ZapStore()],
-        },
-      })
-      expect(wrapper.html().length).toBeGreaterThan(90)
-    },
-    timeout.short()
-  )
```
Comment (brdandu) [inline test/ui.test.js:199]: Why are these 2 tests removed?

File: src/layouts/ZclLayout.vue
Comment (tbrkollar) [reply src/layouts/ZclLayout.vue:1]: we are not using there anymore

File: src/components/ZclCreateModifyEndpoint.vue (hunk 4 above)
Comment (tbrkollar) [reply src/components/ZclCreateModifyEndpoint.vue:61]: in standalone zigbee mode, we could not use the backspace / remove btn on the new card endpoint card. But if we modify this, it is working well. (I had a meeting with @paulr34, @dhchandw, @ethanzhouyc and this was one of the topics.)

File: src/components/ZclClusterView.vue (hunk 2 above)
Comment (tbrkollar) [reply src/components/ZclClusterView.vue:225]: it is not needed anymore. It was necessary for the table's height in popup.

File: src/components/ZclClusterView.vue (hunk 3 above)
Comment (tbrkollar) [reply src/components/ZclClusterView.vue:237]: similar to the previous comment

**Review by brdandu — APPROVED** [review]
